assets/js/modules/analytics/components/dashboard/AnalyticsAdSenseDashboardWidgetLayout.js

```diff
@@ -25,18 +25,22 @@ import { __, _n, sprintf } from '@wordpress/i18n';
  * Internal dependencies
  */
 import Data from 'googlesitekit-data';
-import { MODULES_ADSENSE, DATE_RANGE_OFFSET } from '../../../adsense/datastore/constants';
+import { DATE_RANGE_OFFSET } from '../../../adsense/datastore/constants';
+import { MODULES_ANALYTICS } from '../../../analytics/datastore/constants';
```

Comment:
The date range offset values are currently all the same, but given that this links to Analytics I'm thinking we should probably be using the date range offset from Analytics as well. Let's update this and then we should be good to go here.
```suggestion
import { MODULES_ANALYTICS, DATE_RANGE_OFFSET } from '../../../analytics/datastore/constants';
```

Reply:
Ah yes! Good catch 👍